tests/Credentials/CredentialProviderTest.php

```diff
@@ -897,6 +897,108 @@ public function testEnsuresSourceProfileHasCredentials()
         }
     }
 
```

Comment:
Should there be a test that checks to make sure the `sso` provider is part of the default provicer chain?